### PR TITLE
add navbar and infopage

### PR DESCRIPTION
**Pull request by alinawinter**

### CODE REVIEW

**Review by PReichetanz — COMMENTED** [review]
Really nice work! Just some ideas to discuss, but nothing serious 🚀

File: components/Form/FormQuantitySpecification/index.js [hunk 1]
```diff
@@ -152,10 +180,73 @@ const Output = styled.output.attrs({ type: "range" })`
...
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+  margin: 1.5em;
+`;
+
+const ToolTipText = styled.p`
+  margin: 0;
+`;
```
[the remark this answers is not in the document]
Comment (alinawinter) [reply components/Form/FormQuantitySpecification/index.js:218]: Hmm this is for this particular styling of the text in this box. But you are right, this repetitive, I integrated the "ExplanationToolTip"-styling and the "ToolTipText" into one now.

File: components/WeekdayOptions/WeekdayOptions.js [hunk 2]
```diff
@@ -66,22 +66,27 @@ export default function WeekdayOptions({
...
-  align-items: baseline;
   justify-content: space-around;
+  align-items: baseline;
   width: 100%;
+  gap: 1.5em;
+`;
+
+const StyledHeading = styled.h3`
+  margin-left: auto;
+  margin-right: auto;
```
[the remark this answers is not in the document]
Comment (alinawinter) [reply components/WeekdayOptions/WeekdayOptions.js:77]: Yes! I actually didn't know this attribute!

**Comment by alinawinter** [comment]
Thank you @PReichetanz for helping me make my code look cleaner!!

**Pull request merged**